gdal/frmts/jpegxr/frmt_jpegxr.html

```diff
@@ -0,0 +1,80 @@
+<html>
+<head>
+<title>JPEGXR -- JPEG XR File Format</title>
+</head>
+
+<body bgcolor="#ffffff">
+
+<h1>JPEGXR -- JPEG XR File Format</h1>
+
+<p>The JPEG XR format is supported for reading, and batch writing, but not
+update in place.  JPEG XR files are represented as one band (Greyscale) or three
+band (RGB) or four band (RGB with Alpha) datasets with Byte valued bands.<p>
+
+<p>The driver is implemented based on the 
+    <a href="https://jxrlib.codeplex.com/">jxrlib</a> - open source (BSD)
+    implementation of JPEG XR format by Microsoft.</p>
+
+<h2>Driver Features</h2>
+
+<ul>
+<li>Compression - lossless by default, controlled with <code>QUALITY</code> option.</li>
+<li>Image Structure - entire image is decoded; tiling not yet supported.</li>
+<li>Channel Organization - Grey, RGB, BGR, RGBA, BGRA.</li>
+<li>Color - RBG images YUV with 8-bit per sample and
+<a href="https://en.wikipedia.org/wiki/Chroma_subsampling">4:4:4</a> full chroma
+resolution encoding by default.</li>
+</ul>
+
+
+<p>The driver preserves full chroma resolution 
+
+<h2>Creation Options</h2>
+
+<p>JPEG XR files are created using the "JPEG" driver code.
+ JPEG file creation is implemented by the batch (<code>CreateCopy</code>) method. </p>
+
+<p>Encoder properties can be controlled with the following options:</p>
+
+<ul>
+<li> <b>QUALITY=n</b>: By default the quality flag is set to 100, but this
+option can be used to select other values.  Values must be in the
+range from 0 (lossy) to 100 (lossless).<p></li>
+<li> <b>OVERLAP=[0,1,2]</b>: Overlap processing level
+  <ul>
+    <li>0 - No overlap processing (default for lossless quality)</li>
+    <li>1 - One level (default for lossy quality), modifying 4x4 block encoded values
+        based on values of neighboring blocks.</li>
+    <li>2 - Two level, in addition to the first level, encoded values of 16x16 macro
+        blocks are modified based on the values of neighboring macro blocks.</li>
+  </ul>
+</li>
+</ul>
+
+<h2>Licensing</h2>
+
+<p>Microsoft has patents on the technology in JPEG XR.</p>
+<p>Microsoft grants permission to implement the JPEG XR specification under the 
+    <a href="https://en.wikipedia.org/wiki/Microsoft_Open_Specification_Promise">Microsoft Community Promise</a>.
+</p>
+<p>The <a href="http://jxrlib.codeplex.com">jxrlib</a> - open source implementation of
+JPEG XR is available the BSD License.</p>
+
+<h2>See Also:</h2>
+
+<p>
```

Comment:
The file is a copy of [frmt_jpeg.html](https://github.com/OSGeo/gdal/blob/fc71c9e61c68a90972c01b3e1c97208fd789b03b/gdal/frmts/jpeg/frmt_jpeg.html#L140), which contains

```
<h3>See Also:</h3>
<p>

<ul>
<li>
...
```

I will fix it.